runtime/drivers/duckdb/olap.go

```diff
@@ -122,3 +123,7 @@ func rowsToSchema(r *sqlx.Rows) (*runtimev1.StructType, error) {
 
 	return &runtimev1.StructType{Fields: fields}, nil
 }
+
+func (c *connection) DropDB() error {
+	return os.Remove(c.config.DBFilePath)
```

Comment:
This should probably call `c.Close()` before removing the file – otherwise, it might try to write data back to the file after this call.

This also means `c.Close()` should be idempotent (do nothing if already closed), since it will probably be called afterwards as well by the caller.